addons/l10n_in_ewaybill_stock/models/l10n_in_ewaybill.py

```diff
@@ -0,0 +1,603 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import base64
+import json
+import logging
+import pytz
+import re
+from datetime import datetime
+from collections import defaultdict
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+from odoo.addons.l10n_in_ewaybill_stock.tools.ewaybill_api import EWayBillApi, EWayBillError
+
+
+_logger = logging.getLogger(__name__)
+
+
+class Ewaybill(models.Model):
+    _name = "l10n.in.ewaybill"
+    _description = "e-Waybill"
+    _inherit = ['portal.mixin', 'mail.thread', 'mail.activity.mixin']
+
+    # Ewaybill details generated from the API
+    name = fields.Char("e-Way bill Number", copy=False, readonly=True, tracking=True)
```

Comment:
Didn't get it :/ 

Are you asking to rename the below fields? I better would keep it `ewaybill_*` for the below fields as we storing the response.

name should be kept as it is because we need it as a identifier as well for `ewaybill_id`

Reply:
I mean in the string, the text that the user sees, we are not consistent: sometimes we say E-waybill and here i tis e-Way bill